src/packages/core/property/property/property.element.ts

```diff
@@ -359,6 +369,13 @@ export class UmbPropertyElement extends UmbLitElement {
 				display: block;
 			}
 
+			.not-supported {
```

Comment:
Ideally we should not make this with just a class, instead we should make a component for an alert-message.